### PR TITLE
add readme

### PR DESCRIPTION
**Pull request by pirey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+* Tokenizer PHP Extension
+* XML PHP Extension
+* [Redis](https://redis.io/)
+
+clone project :
+
+* Dengan https :
+
+```
+git clone https://gitlab.com/ajaro-id/simdes/simdes-api.git
```
Comment (andhikayuana) [inline README.md:23]: @pirey iki commande ganti gon github

File: README.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+```
+
+* Dengan ssh :
+
+```
+git clone git@gitlab.com:ajaro-id/simdes/simdes-api.git
```
Comment (andhikayuana) [inline README.md:29]: @pirey iki juga tlg ganti ning github

File: README.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+### Postman Collections
+
+ 1. Import postman collections file di folder `collections` ke dalam postman
+ 2. Buat environment dan tambahkan variable berikut :
+
+    a. `BASE_API_URL` untuk url api, misalnya `http://localhost:8000`
+
+    b. `JWT` untuk token api `Bearer yourtokenheregengs`
+
+### Break down into end to end tests
```
Comment (andhikayuana) [inline README.md:148]: @pirey sections iki :
* Break down into end to end tests
* And coding style tests
* Deployment
* Versioning
* Authors
* Acknowledgments 
dihapuske

**Comment by pirey** [comment]
ngene wes ok rung? @andhikayuana

File: README.md (hunk 1 above)
Comment (andhikayuana) [reply README.md]: Ki tlg @pirey

File: README.md (hunk 2 above)
Comment (andhikayuana) [reply README.md]: Iki @pirey

File: README.md (hunk 3 above)
Comment (andhikayuana) [reply README.md]: Iki @pirey

**Pull request closed without merging**